spec/codeowners/checker/group_spec.rb

```diff
@@ -173,6 +173,15 @@ def add_content(group, text)
         expect(subgroups.map(&:title)).to eq([])
       end
     end
+
+    context 'when a pattern has been added to the end of the main_group' do
+      before { subject.add(pattern) }
```

Comment:
maybe we should suggest creating a new group when adding to main group?

Reply:
Yes, we have #32. Maybe we can do it next? I think we should still have an option for adding it at the end.

Reply:
I agree we can do next.